File: weekOfOct23/powerOfFour.py
class Solution(object):
    def isPowerOfFour(self, n):
        """
        :type n: int
        :rtype: bool
        """

        if n <= 0:
            return False

        log = math.log(n, 4)

        if log % 1 == 0:
            return True
        else:
            return False

File: weekOfOct23/powerOfFour.js
/**
 * @param {number} n
 * @return {boolean}
 */
var isPowerOfFour = function(n) {
  if (n <= 0) return false

  const log = Math.log(n) / Math.log(4)

  if (log % 1 === 0 ) return true;
  else return false;
};